src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/Stepper.tsx

```diff
@@ -25,11 +25,10 @@ enum StepperProcessesSteps {
 }
 
 const ReactStepper = ({ step, registerButtonEnabled, subTextAlign, subText }: StepperProps) => {
-    const stores = useStores();
-    const operatorStore: OperatorStore = stores.Operator;
     const classes = useStyles({ step, registerButtonEnabled, subTextAlign });
+    const operatorFeeData = useAppSelector(getOperatorFeeData);
     // @ts-ignore
-    const time = (operatorStore.operatorApprovalBeginTime - operatorStore?.declaredOperatorFeePeriod) * 1000;
+    const time = (operatorFeeData.operatorApprovalBeginTime - operatorFeeData?.declaredOperatorFeePeriod) * 1000;
```

Comment:
this one can result in NaN, please remove the @ts-ignore, and add a fallback like (operatorStore?.declaredOperatorFeePeriod || 0) or something else

Reply:
+